ci-operator/config/openshift/cluster-logging-operator/openshift-cluster-logging-operator-master.yaml

```diff
@@ -128,7 +128,7 @@ tests:
     product: ocp
     timeout: 1h0m0s
     version: "4.14"
-  skip_if_only_changed: ^docs/|\.adoc$|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
+  skip_if_only_changed: ^docs/|\.adoc$|\.md$|\.tpl$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
```

Comment:
I think this should actually be:
```^docs/|^config/docs|\.adoc$|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)```

Reply:
And if you will need to run `make jobs`.

Reply:
Thanks!

Reply:
running `make jobs` gives me some permission denied errors, fyi